src/Options/AbstractOption.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Onliner\ImgProxy\Options;
+
+abstract class AbstractOption
+{
+    /**
+     * @return string
+     */
+    abstract public function name(): string;
+
+    /**
+     * @return array<mixed>
+     */
+    abstract public function data(): array;
+
+    /**
+     * @return string
+     */
+    public function value(): string
+    {
+        $data = $this->data();
```

Comment:
может сделать array_filter? и не нужен будет `rtrim`?
```suggestion
        $data = array_filter($this->data());
```

Reply:
эм... к сожалению нет. часть опций "внутри" может быть пропущена. например `padding-left` будет `pd::::200` :trollface: